bugbug/db.py

```diff
@@ -115,6 +132,22 @@ def upload(path):
 
 
 def last_modified(path):
+
+    # if database version changed then download the database again
```

Comment:
If the version changed, `last_modified` should return None.

Reply:
Returned None in the next commit.